lms/djangoapps/learner_home/mock_data.json

```diff
@@ -43,7 +44,9 @@
                 "courseId": "course-number0-course-id0"
             },
             "courseProvider": {
-                "name": "edX Course Provider"
+                "name": "edX Course Provider",
+                "website": "www.edx.com",
+                "email": "support@edx.com"
```

Comment:
We don't currently supply these or have access to these

Reply:
...what?